ingestion-lambda/src/handler.ts

```diff
@@ -104,22 +107,13 @@ export const main = async (event: SQSEvent): Promise<SQSBatchResponse> => {
 							}),
 						);
 
-						// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- seems like postgres.js requires this format? https://github.com/porsager/postgres/issues/587#issuecomment-1563262612
-						const snsMessageContent: { keywords?: string | string[] } =
-							safeBodyParse(body);
-
-						const finalContent = Array.isArray(snsMessageContent.keywords)
-							? snsMessageContent
-							: {
-									...snsMessageContent,
-									keywords: processKeywords(snsMessageContent.keywords), // re-write the keywords field as an array
-								};
```

Comment:
have we lost the `processKeywords` call here?

Reply:
the logic has been relocated to `OptionalStringOrArrayAsArrayOfStrings` in the Zod schema file. I'm in two minds about locating custom parsing logic in Zod though, so am happy to discuss 🙂